simpletransformers/classification/classification_model.py

```diff
@@ -123,6 +123,10 @@ def __init__(
         }
 
         self.args = self._load_model_args(model_name)
+        
+        if self.args.thread_tune:
+            torch.set_num_threads(1) 
```

Comment:
Assuming `thread_tune` is renamed to `thread_count` and the datatype changed to `int`, this would then be:

`torch.set_num_threads(self.args.thread_count)`

Reply:
Made the suggested change